include/aikido/trajectory/util.hpp

```diff
@@ -0,0 +1,95 @@
+#ifndef AIKIDO_TRAJECTORY_UTIL_HPP_
+#define AIKIDO_TRAJECTORY_UTIL_HPP_
+
+#include "aikido/trajectory/Interpolated.hpp"
+#include "aikido/trajectory/Spline.hpp"
+
+namespace aikido {
+namespace trajectory {
+
+/// Converts an interpolated trajectory to a piecewise linear spline trajectory.
+///
+/// This function requires the \c _inputTrajectory to use a \c
+/// GeodesicInterpolator. So the conversion exactly preserves the geometric
+/// path.
+///
+/// \param[in] inputTrajectory Interpolated trajectory
+/// \return A spline trajectory
+UniqueSplinePtr convertToSpline(const Interpolated& inputTrajectory);
+
+/// Converts a piecewise linear spline trajectory to an interpolated trajectory
+/// using a given interpolator.
+///
+/// \param[in] traj Spline trajectory
+/// \param[in] interpolator Interpolator used in connecting two ends of a
+/// segment
+/// \return An interpolated trajectory
+UniqueInterpolatedPtr convertToInterpolated(
+    const Spline& traj, statespace::ConstInterpolatorPtr interpolator);
+
+/// Concatenates two interpolated trajectories.
+///
+/// This function concatenates two interpolated trajectories into one
+/// interpolated trajectory. The start state of the last segment in the first
+/// trajectory is connected with the start of the first segment in the second
+/// trajectory in concatenation. For example, concatenating trajectory a:
+/// [wp1(t=1), wp2(t=2.1), wp3(t=4)] and trajectory b: [wp4(t=2), wp5(t=7)]
+/// gets a new trajectory: [wp1(t=0), wp2(t=1.1), wp4(t=4), wp5(t=9)].
```

Comment:
I think this implementation keeps the original start time of trajectory a? So I think it should look like

```suggestion
/// gets a new trajectory: [wp1(t=1), wp2(t=2.1), wp4(t=4), wp5(t=9)].
```

Reply:
That's correct. I thought it's updated so but wasn't. :rage: Thanks!